src/Impostor.Server/Net/Inner/Objects/Components/InnerPlayerPhysics.cs

```diff
@@ -27,12 +27,12 @@ public InnerPlayerPhysics(ICustomMessageManager<ICustomRpc> customMessageManager
 
         public override ValueTask<bool> SerializeAsync(IMessageWriter writer, bool initialState)
         {
-            throw new NotImplementedException();
+            return new ValueTask<bool>(false);
         }
 
         public override ValueTask DeserializeAsync(IClientPlayer sender, IClientPlayer? target, IMessageReader reader, bool initialState)
         {
-            throw new NotImplementedException();
+            return ValueTask.CompletedTask;
```

Comment:
Does it make sense to add an anticheat check here? Vanilla doesn't send out these kinds of packets

Reply:
Perhaps we are supposed to add a check to set whether or not to run Vanlia's unused correlation